.travis.yml

```diff
@@ -25,10 +25,12 @@ matrix:
       env: DISTRO=ubuntu DISTRO_VERSION=17.04 CHECK_STYLE=yes GENERATE_DOCS=yes
     - os: linux
       compiler: clang
-      env: DISTRO=ubuntu DISTRO_VERSION=16.04 SANITIZE_ADDRESS=yes BUILD_TYPE=Debug
+      env: DISTRO=ubuntu DISTRO_VERSION=16.04 BUILD_TYPE=Debug \
+           CMAKE_FLAGS=-DSANITIZE_ADDRESS=yes
     - os: linux
       compiler: clang
-      env: DISTRO=ubuntu DISTRO_VERSION=16.04 SANITIZE_UNDEFINED=yes BUILD_TYPE=Debug
+      env: DISTRO=ubuntu DISTRO_VERSION=16.04 \
+           CMAKE_FLAGS=-DSANITIZE_UNDEFINED=yes BUILD_TYPE=Debug
```

Comment:
Put `BUILD_TYPE=Debug` on the previous line for consistency?

Reply:
Done.  I may reorder all the environment variables in a future commit because they most interesting stuff is at the end, and therefore less visible in the Travis UI.